Fix link in `CHANGELOG.md`, add `swift sdk install --checksum` item (#7935)
swiftlang/swift-package-manager: CHANGELOG.md

  `--experimental-swift-sdks-path` options on `swift build` are deprecated with replacements that don't have the
  `experimental` prefix.

* [#7722]

>   For `swift sdk install` subcommand passed remote URLs as arguments, an additional `--checksum` option is now required, through which users of a Swift SDK can specify a checksum provided by

Reviewer: A bit rephrased first bit: `An additional `--checksum` option  is now required for `swift sdk install` subcommand with remote URLs as arguments. `--checksum` allows users of a Swift SDK to specify a checksum provided by the SDK publisher.

>   For `swift sdk install` subcommand passed remote URLs as arguments, an additional `--checksum` option is now required, through which users of a Swift SDK can specify a checksum provided by
>   a publisher of the SDK. The latter can produce a checksum by running `swift package compute-checksum` command (introduced in [SE-0272]) with the Swift SDK bundle archive as an argument.

Author: ```suggestion
  An additional `--checksum` option  is now required for `swift sdk install` subcommand with remote URLs as arguments. `--checksum` allows users of a Swift SDK to specify a checksum provided by the SDK publisher. A checksum can be produced by running `swift package compute-checksum` command (introduced in [SE-0272]) with the Swift SDK bundle archive as an argument.
```
